Source/WebCore/page/FrameView.cpp

```diff
@@ -2446,7 +2446,7 @@ void FrameView::scrollToFocusedElementInternal()
 
     bool insideFixed;
     LayoutRect absoluteBounds = renderer->absoluteAnchorRectWithScrollMargin(&insideFixed);
-    FrameView::scrollRectToVisible(absoluteBounds, *renderer, insideFixed, { m_selectionRevealModeForFocusedElement, ScrollAlignment::alignCenterIfNeeded, ScrollAlignment::alignCenterIfNeeded, ShouldAllowCrossOriginScrolling::No });
+    FrameView::scrollRectToVisible(absoluteBounds, *renderer, insideFixed, { m_selectionRevealModeForFocusedElement, ScrollAlignment::alignCenterIfNeeded, ScrollAlignment::alignCenterIfNeeded, ShouldAllowCrossOriginScrolling::No, ScrollBehavior::Auto, renderer->absoluteAnchorRect(&insideFixed) });
```

Comment:
I prefer using a local variable to hold the result of `renderer->absoluteAnchorRect(&insideFixed)`; it makes it easier to debug, and you can name it.

We're also now calling `absoluteAnchorRectWithScrollMargin` and `absoluteAnchorRect` both of which are going to do ancestor traversals. Maybe we should modify `absoluteAnchorRectWithScrollMargin` to return both rects.

Reply:
Fixed.